source/accessibility/index.html.md.erb

```diff
@@ -63,14 +61,8 @@ This [Technical Documentation Template](https://github.com/alphagov/tech-docs-te
 
 DAC tested the [GovWifi technical documentation](https://docs.wifi.service.gov.uk/), which uses this Technical Documentation Template.
 
-This Technical Documentation Template is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard due to the following non-compliance.
-
-When a user searches, the search results dialogue hides the main page region from assistive technology. This fails [WCAG 2.1 success criterion 3.2.2 - On Input](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html) and [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).
-
-We plan to fix this issue by 30 September 2021.
-
-While GDS takes steps to ensure the Technical Documentation Template is as accessible as possible by default, you still need to carry out contextual research.
+This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.
```

Comment:
I think it's probably best to remove the paragraphs on lines 62-64 as well. Otherwise it might sound like the TDT was compliant when DAC tested it in April.